### ISSUE TITLE
Watch folder + auto-tag

### ISSUE THREAD

**Issue by slevey087** [burtonator/polar-bookshelf#1136]
**DONATE OR PURCHASE POLAR PREMIUM FOR PRIORITIZED SUPPORT AND FEATURES **

**DONATE OR PURCHASE POLAR PREMIUM FOR PRIORITIZED SUPPORT AND FEATURES **

https://opencollective.com/polar-bookshelf

**Is your feature request related to a problem? Please describe.**
I keep a few different folders for different formats (eg. books, journal articles, book reviews, newspaper articles, etc), and each time have to manually import then tag them appropriately. It's a pain :)

**Describe the solution you'd like**
POLAR would watch those folders, then as soon as a file is added to one, import it. The folder would be associated with one or more tags, and when the file is imported, automatically tag it. So, to import a book and label it "Book," all I'd have to do is save it in the folder (and open POLAR if it's not already opened).

**Comment by burtonator** [comment]
Yes... other users have asked for this and we will probably add it in the future but might be a premium feature.

**Comment by burtonator** [comment]
This might be harder on the new electron build but I think we should do it.
